include/xcshell/parser.h

```diff
@@ -0,0 +1,22 @@
+#ifndef INCLUDE_XCSHELL_PARSER_H_
+#define INCLUDE_XCSHELL_PARSER_H_
+
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "build_in/build_in.h"
+#include "xcshell/CommandParseResult.h"
+
+class Parser {
+ public:
+  Parser() = default;
+  CommandParseResult ParseUserInputLine(const std::string &input_line);
+  static std::vector<char *> BuildArgv(const std::string &command,
```

Comment:
Do we need this method public? or just private?

Reply:
the method was used by command_executor.cc , i changed two method private , it will report error